app/src/main/res/layout/fragment_profile.xml

```diff
@@ -94,11 +94,79 @@
         app:layout_constraintTop_toBottomOf="@+id/displayNameTv"
         tools:text="example@example.com" />
 
+    <com.google.android.material.card.MaterialCardView
+        android:id="@+id/premiumCv"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_margin="20dp"
+        app:cardCornerRadius="24dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/emailTv">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:background="@drawable/rounded_cv_bordered_red"
+            android:padding="25dp">
+
+            <ImageView
+                android:id="@+id/premiumIv"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:contentDescription="@string/app_name"
+                android:src="@drawable/ic_premium"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
+
+            <LinearLayout
```

Comment:
nit: Next time please do not nest viewGroups if you can avoid it. Above constraintLayout is enoguh to build this UI

Reply:
I could not handle the situation, When you said I remembered about chainStyle (spread, packed), 
was it the right way to do so?